pkg/operator/ceph/csi/spec.go

```diff
@@ -213,12 +217,23 @@ func StartCSIDrivers(namespace string, clientset kubernetes.Interface, ver *vers
 			return fmt.Errorf("failed to start rbdplugin daemonset: %+v\n%+v", err, rbdPlugin)
 		}
 	}
+
+	tolerations := getToleration()
+	nodeAffinity := getNodeAffinity()
 	if rbdProvisionerSTS != nil {
+		rbdProvisionerSTS.Spec.Template.Spec.Tolerations = tolerations
```

Comment:
Are these properties only applied to the provisioners? We need to also apply them to the plugins above. But I would also argue that the user might want to apply different node affinity or tolerations for the provisioners vs drivers. For example, the provisioners might go on nodes with other Ceph daemons, but the drivers need to go wherever the clients are running.

Reply:
@travisn do we need to apply these to node-plugins ( which runs as daemonsets)? i assume this should run on all the nodes where the users what to run workloads on

Reply:
Right, the daemonsets can take node affinity and tolerations as well. For example, the client workloads might need tolerations depending on the taints in the cluster. But did that make sense that we need different properties for these since they could be different from the provisioners' affinity and tolerations?